### DISCUSSION TITLE
【翻訳案】DLC初出の単語

### DISCUSSION THREAD

**Discussion by nsk4762jp** [nsk4762jp/OxygenNotIncluded-Japanese#116] [Ideas]
DLC初出の単語について、この場でいろいろ検討できたらなと思います。
まだ一覧化できてませんが、随時リストアップしていく予定です。

原語 | 翻訳案 | 備考
----- | ----- | -----
Beetasfinery | ビータの巣 | finery はfine「美しい」-ery「集合体」の意
Grubfruit | グラブフルーツ | 
Spindly Grubfruit | ひょろひょろグラブフルーツ |  
Radiation | 放射線 |
Trailblazer Lander | 先遣隊着陸船 |  
Trailblazer Module | 先遣隊モジュール | 
Plug Slug | びりびりナメクジ | #158 の提案より
Grubgrub | グラブグラブ | Grubfruitの成熟に必要
Sweetle | スウィートル |  Grubfruitの成熟に必要
Divergent | ダイバージェント | Grubgrub, Sweetle の属
Saturn Critter Trap | ケモノジゴク | Venus Flytrap(ハエトリグサ、ハエジゴク)の捩り
Hydrocactus | ミズサボテン | Hydro「水の」 -cactus「サボテン」
Bog Bucket | ドロバケツ | Bog, Swamp の違いを出せたらいいなあ
Bog Jelly | 泥炭地ゼリー | 
Swampy Delights | 泥炭地の楽しみ | 
Tranquil Toe | アシモミ | DESCの記述から造語。footrub はつま先マッサージのことらしい。
Mellow Mallow | メロマロ | Mellow「芳醇な」 Mallow「アオイ」
Swamp Chards | ヌマビーツ  | Swamp「沼」Chards「[フダンソウ](https://ja.wikipedia.org/wiki/%E3%83%95%E3%83%80%E3%83%B3%E3%82%BD%E3%82%A6) (ビートの一種)」
Swamp Chard Heart | ヌマビーツの結球 | 
World | 惑星 | 「世界」ではない気がする 
Interior Rocket Outlets | 電源コンセント |  
Radbolt | RADボルト | kankoromochi さんの Radbolt 説明文から着想  
Refined Carbon | コークス | 
Flight/Travel | 航行 | denev-rove さんのプルリクから 
Flight Path | 航行路 | 
Fitting | 端子 | 
Gantry | 整備用足場 | 
Rover | 探査ロボ |

**Comment by halver3214** [comment]
GrubfruitとSpindly Grubfruitについて
とりあえず「グラブフルーツ」と「ひょろひょろグラブフルーツ」としました。
どちらも種子は同じなのですが、生物("Sweetle"と"Grubgrub"の両方)が植物の世話をすることで
良質な実(Grubfruit)をつけるというシステムのようです。
grubには「[noun]昆虫(特に甲虫)の幼虫」と「[verb]掘り起こす」という意味があるようです。

**Reply by nsk4762jp** [reply]
ありがとうございます。一覧に追加しておきました。
世話されない自然の状態だと Spindly、世話されると Grubfruit ですね。  
ひょろひょろのほうは熟してないのかと思いきや、熟してもなおひょろひょろってことですか。

**Reply by halver3214** [reply]
その通りのようです。
ゲーム内で栽培してみたのですが、普通に複製人間(農家スキル持ち)が世話をしてもSpindly Grubfruitのまま成熟します。
2種類の昆虫(緑色の芋虫"Grubgrub"と藍色のコガネムシ"Sweetle")が植物にアレコレすることでGrubfruitに変化します。
片方だけではダメのようです。

**Comment by nsk4762jp** [comment]
現実の宇宙探査船やローバーには「パイオニア」だとか「パスファインダー」とかカッコいい名前が付いてるんですが、ゲーム中にも Trailblazer と言う名前で出てきました。これをそのまま翻訳しても良いものか悩んでいます。
今のところ暫定で別の語を割り当てていますが何か良いアイデアがあればお願いします。

**Reply by Halo-Kitty** [reply]
Trailblazerを辞書で引くと先駆者、開拓者と出ますね
「トレイルブレイザー」って自動車名でも発売されてるので、そのままでもいいかもしれません
余談ですが先行者にすると中国の昔流行ったロボットみたいですねｗ

**Reply by nsk4762jp** [reply]
そのまま当てはめると、UI から文字がはみ出そうなんですよね。 `トレイルブレイザー モジュール`  
ぎりぎり納まるかな？

**Reply by Halo-Kitty** [reply]
文字制限に引っかかるようなら、あくまで一案ってことで参考まで
「トレイルブレイザ」「ー」除去
「トレイブレイザ」米語風に「ル」を発音しない
「ブレイザー」だけで未知を切り開くって意味もあるので前半削除
あとは日本語で「開拓者」ですかねー

**Reply by nsk4762jp** [reply]
`開拓者` は短くまとまってていいですね。それにしましょう。

**Reply by denev-rove** [reply]
いまさらですが、Trailblazer の訳について考えてみました。
実際にゲームを遊んでみた感覚としては、開拓者（定住する拠点を確立する者）というよりは、偵察部隊（一番乗りで現地入りして地形を把握し、後続の部隊に情報を渡す者）といった性質のほうが強い使い方のように思いました。
というわけで「先遣隊モジュール」という訳はどうでしょうか。

**Reply by nsk4762jp** [reply]
denev-rove さんとの会話により、とりあえず下記のとおりとします。

`Trailblazer Module` → `先遣隊モジュール`
`Rover's Module` → `探査ロボモジュール`
`Rover Bot` → `探査ロボ`
`Trailblazer Lander` → `先遣隊着陸船`
`Rover's Lander` → `探査ロボ着陸船`

**Comment by halver3214** [comment]
DLCのゲーム内呼称は、英語の綴りのまま"Spaced Out!"で良いでしょうか。

**Reply by nsk4762jp** [reply]
これまでの xxx upgrade の時も一応翻訳はしたので、今回は「宇宙進出」とでも訳しておきましょう。

**Reply by nsk4762jp** [reply]
一年前のコメントであれですが、やっぱり原文のまま "Spaced Out!" にしようと思います。

**Comment by nsk4762jp** [comment]
Steamの翻訳スレッドで [Halo Kitty さんから訳語の提案](https://steamcommunity.com/workshop/filedetails/discussion/928606476/1290690669224436955/?tscn=1608432320) がありました。

> こんにちは
が翻訳して下さったITEMSで気になった箇所があったので、私なりの翻訳を載せておきます
"Food smeared on the wall from a recent Food Fight"
最近食べ物をぶつけ合った時の壁の食べ物のシミ
最近食べ物で悪ふざけした時の壁の食べ物のシミ
*フードファイト＝大食い選手権という和製英語が有るせいで分かりづらくなっちゃいますね
> 
> "Food Splatter"
> 食べこぼし
> *Food Fightの結果の飛び散りなので「飛び散った食べ物」という事でしょう。説明っぽいから「食べこぼし」にしました
> 
> "A disturbingly delicious mixture of irradiated dirt and water."
> 放射線照射された土と水でできた心をかき乱すほど美味しい混合物です。
> *disturbingly deliciousでとても美味しいものを指す慣用句のようです
> 
> 個人的にはhalver3214さんのレンチンレタスが秀逸だと思いました

**Reply by nsk4762jp** [reply]
以下は該当スレッドに投稿した僕のコメントです。

---
どうもありがとうございます。GitHubのほうのディスカッションにも登録しました。
このゲームのフードファイトは食べ物を投げ合うイベントみたいですね。

> STRINGS.GAMEPLAY_EVENTS.EVENT_TYPES.FOOD_FIGHT.DESCRIPTION
> Duplicants will throw food at each other for recreation
> 
> It may be wasteful, but everyone who participates will benefit from a major stress reduction.

「最近食べ物～時の壁の食べ物のシミ」で少し冗長な気がするので「最近食べ物～時にできた壁のシミ」にしようと思います。

"Food Splatter" の「食べこぼし」ですが、食べ物を投げ合っているので(食べる用途ではないので)「食べ物の残骸」でいいかなと思います。

"A disturbingly delicious mixture of irradiated dirt and water." 放射線照射された土と水でできた心をかき乱すほど美味しい混合物です。
これは取り込んでおこうと思います。(ホントに美味しいかどうかは怪しいですね 😆 )

**Comment by nsk4762jp** [comment]
Reactor の翻訳について、(自分の中でも)訳語の揺らぎがあるので備忘録として書いておくことにします。

Nuclear Reactor → 原子炉
Research Reactor → 研究炉
Reactor → 原子炉

**Comment by Halo-Kitty** [comment]
DLCの新要素ではなく、何ヶ月も決まった翻訳なので余計な事かもしれませんが
”vertical wind tunnel” の翻訳が「垂直型風洞」になっているのに疑問を感じました
”vertical wind tunnel” はアクティビティの名称なので、「垂直型風洞」と状態を表すより
Wikipediaや日本の同様のレジャー施設で使われている「インドアスカイダイビング」の名称が分かりやすく、ふさわしいのでは？
そう思った次第です、単なる提案なのでご迷惑なら聞き流してください

**Reply by nsk4762jp** [reply]
どうもです。

施設 `vertical wind tunnel` を `垂直型風洞` と訳したのは[辞書に載っていた](https://eow.alc.co.jp/search?q=vertical+wind+tunnel)のと[そういう使用例があった](https://english.cheerup.jp/article/5124?page=1)ためで、特に深く考えて訳したわけではないです。  
確かにやってることはインドアスカイダイビングなのでこっちでもいいかもしれませんね。

**Reply by Halo-Kitty** [reply]
早速のご返信ありがとうございます！
インドアスカイダイビングは日本でも新しいので、まだ辞書も直訳みたいですね

ところでRadbolt from the Blue Updateの翻訳をお手伝い出来ないかと、見ておりました
どこに手を出していいのかとGitHubの使い方もまだわかっていないので、ご指示いただけましたらその項目をやらせていただきます

**Reply by nsk4762jp** [reply]
分量が少ない RESEARCH カテゴリ辺りはどうでしょうか(翻訳箇所17個)。
[GitHub Desktop](https://desktop.github.com/) と [POEdit](https://poedit.net/) があれば多分すぐ始められると思います。
GitHub から GitHub Desktop にリポジトリを持ってくる方法は[ここ](https://docs.github.com/ja/desktop/contributing-and-collaborating-using-github-desktop/cloning-a-repository-from-github-to-github-desktop)や[ここ](https://docs.github.com/ja/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop)が参考になると思います。

**Reply by Halo-Kitty** [reply]
頂いたリンク先を見ておりました
最初に翻訳するファイルを引っ張ってこないと行けないのかな
まだツールの使い方がわからないので、追いつけないようならそのまま進めてください
すみません

**Reply by denev-rove** [reply]
”vertical wind tunnel”について私見を述べさせてください。
分かりやすさを優先するなら、インドアスカイダイビングは素晴らしい訳だと思います。（何をする施設なのかすぐ分かるので）
ただ、”vertical wind tunnel”はあくまで施設の名称だと思います。
インドアスカイダイビングはアクティビティの名称なので、「垂直型風洞」（つまり、風洞という施設）も決して悪い訳ではないと思います。（Wikipediaによれば中国語でも「垂直風洞」というようです）
直訳ながら、なかなか風情のある訳だと思っていたので、再検討をお願いしたいです。

仮に分かりやすさを優先する場合でも、カタカナ12文字はかなり長いと感じます。
「屋内スカイダイビング」というのも一案かなと思います。（それでも10文字ですが）

個人的意見としては、元の「垂直(型)風洞」、次点で「屋内スカイダイビング」を推します。
ご意見よろしくおねがいします。

**Comment by nsk4762jp** [comment]
電化製品の消費電力を測るアイテムを「ワットチェッカー」と呼び、「ワット数センサー `Wattage sensor`」がこれに該当するのではないか、という提案を受けました。  
電器にはあまり詳しくないので、有識者の意見を伺いたいです。
